dynamodb.tf

```diff
@@ -1,7 +1,7 @@
 module "dynamodb_table" {
   source = "git@github.com:xerris/aws-modules.git//dynamodb"
 
-  name      = "${var.env}-${var.dynamodb_table_name}"
+  name      = "${var.env}-${var.dev-project-terraform-state-lock-dynamo}"
```

Comment:
this should be a different dynamo table than terraform state table

Reply:
Shall I put it as dynamodb_table_nithin ?